scripts/onboard-new-task-with-ci-stub.sh

```diff
@@ -12,6 +12,13 @@ if ! echo "${TASK_NAME}" |  grep -qE "^[a-z-]+$"; then
   exit 1
 fi
 
+read -p "Is it only OKD task? true/false: " ONLY_OKD
+
+if ! echo "${ONLY_OKD}" |  grep -qE "^true$|^false$"; then
+  echo "Invalid bool value! Should comply with ^true$|^false$ regex" 1>&2
+  exit 1
+fi
+
 read -p "What is the name of the env variable for this task: " TASK_ENV_VAR
```

Comment:
IMO I would put the OKD question behind the env variable one. To not to switch the context and let the user name the env var in the same fashion as the task name :)